content/es/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -33,7 +33,7 @@ Kubernetes es un proyecto que soporta y mantiene los controladores Ingress de [A
 * [Contour](https://projectcontour.io/) es un controlador Ingress basado en [Envoy](https://www.envoyproxy.io/).
 * [Emissary-Ingress](https://www.getambassador.io/products/api-gateway) es un API Gateway [Envoy](https://www.envoyproxy.io)-basado en el controlador Ingress.
 * [EnRoute](https://getenroute.io/) es un API Gateway basado en [Envoy](https://www.envoyproxy.io) que puede correr como un controlador Ingress.
-* [Easegress IngressController](https://github.com/megaease/easegress/blob/main/doc/reference/ingresscontroller.md) es una API Gateway basada en [Easegress](https://megaease.com/easegress/) que puede correr como un controlador Ingress.
+* [Easegress IngressController]((https://megaease.com/docs/easegress/04.cloud-native/4.1.kubernetes-ingress-controller/) es una API Gateway basada en [Easegress](https://megaease.com/easegress/) que puede correr como un controlador Ingress.

```

Comment:
The change requires a minor correction, as it is slightly off.

```suggestion
* [Easegress IngressController](https://megaease.com/docs/easegress/04.cloud-native/4.1.kubernetes-ingress-controller/) es una API Gateway basada en [Easegress](https://megaease.com/easegress/) que puede correr como un controlador Ingress.

```

Reply:
Done, thanks